search.php

```diff
@@ -8,13 +8,13 @@
 
 get_header();
 ?>
-<section id="primary" class="content-area">
-	<main id="main" class="site-main" role="main">
+<section id="primary" class="entry__area">
+	<main id="main" class="site__main" role="main">
 
 	<?php if ( have_posts() ) : ?>
 
-		<header class="page-header">
-			<h1 class="page-title">
+		<header class="page__header">
```

Comment:
Both elements does not have any relationship as they are named here can you make sure it's named properly.